### ISSUE TITLE
201801416_Challenges_Solutions

### ISSUE THREAD

**Issue by khyatibhuva** [nisargptl/vehicle-recognition#72]
Write your challenges you faced in this entire project and how to solved them.

**Comment by Mohil027** [comment]
Challenges :  1) For our project we found a dataset prepared by the Stanford University AI Lab consisting of the images for the cars, but the other important details of the cars except it’s names were not available anywhere. The cars in this database were all manufactured during 2012 or before that period, thus, it became very difficult to find the details of the cars. 
Solution 1 : The best possible option to make an accurate database was none other than doing the manual search. So we manually collected the data from official websites for all the features of the cars.
Also, there were many issues while running queries which required frequent changes in the database in order to solve those issues in a pragmatic approach.

2) For the image dataset, all the car images were required in a specific pixel dimensions format i.e. (800 x 305). But while searching images, none of the images were specifically found in that dimensions.
Solution : All the images had to be resized and formatted according to the required dimensions format.
